redcap/methods/surveys.py

```diff
@@ -11,6 +11,58 @@
 class Surveys(Base):
     """Responsible for all API methods under 'Surveys' in the API Playground"""
 
+    def export_survey_link(
+        self,
+        instrument: str,
+        record: str,
+        event: Optional[str] = None,
+        repeat_instance: int = 1,
+    ):
+        """
+        Export one survey link
+
+        Note:
+            The passed instrument must be set up as a survey instrument.
+
+        Args:
+            instrument:
+                Name of instrument as seen in the Data Dictionary (metadata).
+            record:
+                Name of the record
+            event:
+                Unique event name, only used in longitudinal projects
+            repeat_instance:
+                only for projects with repeating instruments/events) 
+                The repeat instance number of the repeating event (if longitudinal) 
+                or the repeating instrument (if classic or longitudinal). 
+                Default value is '1'.
+
+        Returns:
+            Str:
+                URL of survey Link requestet
+
+        Examples:
+            >>> proj.export_survey_link(instrument="form_1", record="5", event="event_1_arm_1")
+            https://redcap.mytld.com/surveys/?s=6B2zFAEWPVSrXnnx
```

Comment:
I actually have a live project on redcapdemo.vumc.org for to run `doctest` on docstring examples -- I'll just change this after the merge since that'll be easiest 

Just need to remember lol 🎗️